pkgs/applications/networking/instant-messengers/armcord/default.nix

```diff
@@ -0,0 +1,130 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, makeWrapper
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, ffmpeg
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gtk3
+, libappindicator-gtk3
+, libdbusmenu
+, libdrm
+, libnotify
+, libpulseaudio
+, libsecret
+, libuuid
+, libxkbcommon
+, mesa
+, nss
+, pango
+, systemd
+, xdg-utils
+, xorg
+}:
+
+with lib;
+stdenv.mkDerivation rec {
+  inherit (stdenv.hostPlatform) system;
+
+  pname = "armcord";
+  version = "3.0.7";
+
+  x86_64-linux-sha256 = "b2a583e6abbc6e5dc3f7370a33f21fc4e7963c6cbe7555e954156c77e9577261";
+  aarch64-linux-sha256 = "8c32a14ab8e5bdf865a6523cb4b5cec8f3f870b95f99be9661a4dd0df33aae1d";
+
+  src = let
+    base = "https://github.com/ArmCord/ArmCord/releases/download";
+  in {
+    x86_64-linux = fetchurl {
+      url = "${base}/v${version}/ArmCord_${version}_amd64.deb";
+      sha256 = x86_64-linux-sha256;
+    };
+    aarch64-linux = fetchurl {
+      url = "${base}/v${version}/ArmCord_${version}_arm64.deb";
+      sha256 = aarch64-linux-sha256;
+    };
+  }.${system};
```

Comment:
I think an 'or throw <not supported>' needs to be here. I think due to the way platforms is set it might be ok, but let's be consistent.

Reply:
Done. For some reason it was giving me trouble (and was removed because of the platforms bit) before, but now works fine.